platform/ios/CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ Mapbox welcomes participation and contributions from everyone. Please read [CONT
 
 * Fixed a custom geometry source bug caused by using the outdated tiles after style update [#15112](https://github.com/mapbox/mapbox-gl-native/pull/15112)
 
+### Offline maps
```

Comment:
The Mobile Cache Management API affects more than just offline maps, because it allows for control over the ambient cache used in regular (non-offline) maps. Do the iOS release notes ever use "Features" as a heading? I think that would be the most appropriate heading.

Reply:
Let's do it.

Reply:
We don't use features as a heading, but I can move the entry to `Other Changes`. `Features` could be confusing, since we classify things based on what they do, rather than whether they are a feature or a bug fix.